modules/sso.php

```diff
@@ -746,7 +865,42 @@ function allowed_redirect_hosts( $hosts ) {
 		return array_unique( $hosts );
 	}
 
-	function button( $args = array() ) {
+	/**
+	 * Builds the "Login to WordPress.com" button that is displayed on the login page as well as user profile page.
+	 *
+	 * @param  array   $args       An array of arguments to add to the SSO URL.
+	 * @param  boolean $is_primary Should the button have the `button-primary` class?
+	 * @return string              Returns the HTML markup for the button.
+	 */
+	function build_sso_button( $args = array(), $is_primary = false ) {
+		$url = $this->build_sso_button_url( $args );
+		$classes = $is_primary
+			? 'jetpack-sso button button-primary'
+			: 'jetpack-sso button';
+
+		return wp_kses(
```

Comment:
I agree that we could probably simplify it a bit.

That being said, I also think it's probably fine as is for now. I changed it to this on a [suggestion](https://github.com/Automattic/jetpack/pull/3675#discussion_r60479909) from @eliorivero  and it looks similar to the [i18n docs for HTML in strings](https://codex.wordpress.org/I18n_for_WordPress_Developers#HTML).

I'd lean towards getting this PR in and addressing simplification in a follow up issue/PR if we want to.